.github/workflows/docker_build_deploy.yml

```diff
@@ -32,7 +32,7 @@ jobs:
 
       - name: Create version.json
         run: |
-          echo "{\"commit\":\"$GITHUB_SHA\",\"version\":\"$GITHUB_REF\",\"source\":\"https://github.com/$GITHUB_REPOSITORY\",\"build\":\"$GITHUB_RUN_ID\"}" > version.json
+          echo "{\"commit\":\"$GITHUB_SHA\",\"version\":\"$GITHUB_REF_NAME\",\"source\":\"https://github.com/$GITHUB_REPOSITORY\",\"build\":\"$GITHUB_RUN_ID\"}" > version.json
```

Comment:
Looking at https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables, it looks like this will change from `/refs/heads/main` to `main`, and `/refs/tags/<tag>` to `<tag>`. This looks like a good change for tags 👍

Reply:
Yea I was hoping this would make it more readable